### PR TITLE
Add prop-types and fix usePic's expected value. Closes #6

### LINKED ISSUE #6: usePic={false} shows user pic instead of hiding.
Passing `usePic={false}` shows the user pic instead of hiding it.
I wonder why are you using "not operator(!)" here?
https://github.com/hprobotic/react-telegram-login/blob/9d2bc5f66b92c4f1ca42fb0d2b6b52e42abd2d19/src/index.js#L19

I can fix this and add prop types but was hesitant because I saw you used `// @flow`. Although, you're not actually using flow and its package is not included in the package.json.

So if you don't mind I'll fix this issue and also add prop-types.

### PR DESCRIPTION
**Pull request by poeti8**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -13,10 +14,10 @@ class TelegramLoginButton extends React.Component {
...
     const script = document.createElement('script');
     script.src = 'https://telegram.org/js/telegram-widget.js?2';
-    script.setAttribute('data-telegram-login', botName || 'samplebot');
-    script.setAttribute('data-size', dataSize || 'large');
-    script.setAttribute('data-request-access', requestAccess || 'write');
-    script.setAttribute('data-userpic', !usePic);
+    script.setAttribute('data-telegram-login', botName);
+    script.setAttribute('data-size', dataSize);
+    script.setAttribute('data-request-access', requestAccess);
+    script.setAttribute('data-userpic', !!usePic);
```
Comment (hprobotic) [inline src/index.js]: Can we use `usePic` instead `!!usePic` here?

File: src/index.js [hunk 2]
```diff
@@ -36,4 +37,20 @@ class TelegramLoginButton extends React.Component {
...
+  dataSize: PropTypes.oneOf(['large', 'medium', 'small']),
+  requestAccess: PropTypes.string,
+  usePic: PropTypes.bool,
+};
+
+TelegramLoginButton.defaultProps = {
+  botName: 'samplebot',
+  dataOnauth: () => undefined,
+  dataSize: 'large',
+  requestAccess: 'write',
```
Comment (hprobotic) [inline src/index.js:52]: We only need define default props for props that `isRequired` right?

**Review by hprobotic — COMMENTED** [review]
Just put some more comments. It's almost done.

File: src/index.js (hunk 1 above)
Comment (poeti8) [reply src/index.js]: Actually, yes now. Because I added default props so we won't have undefined.

File: src/index.js (hunk 2 above)
Comment (poeti8) [reply src/index.js:52]: No, for those who are **not** required. But nothing stops the user from not entering the required props, so I'd often define default values for the default props too.

**Comment by hprobotic** [comment]
Thanks, look good man 👍

**Review by hprobotic — APPROVED** [review]

**Pull request merged**